rentalPrice.js

```diff
@@ -1,11 +1,13 @@
 // age - age of driver
-// licence - number of full years person holds driving licence
+// expirienceOfDriving - number of full years person has been driving
 // clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
-// acc - has s/he caused any accidents within last year
-// acc2 - has s/he participated (but not caused) in any accidents within last year
+// hadAccidents - has s/he caused any accidents within last year
+// hadIncludedInAccidents - has s/he participated (but not caused) in any accidents within last year
 // season - if it is high season or not
-function price(age, licence, clazz, acc, acc2, season) {
+function price(age, expirienceOfDriving, clazz, hadAccidents, hadIncludedInAccidents, season) {
```

Comment:
Just renaming couple of variables is not really the best way to refactor. 
In addition to renaming, code should be more readable and understandable to the user. Code also needs to be checked that it follows the business rules.